diff -urN base/meta-raspberrypi/recipes-kernel/linux/files/container.cfg modified/meta-raspberrypi/recipes-kernel/linux/files/container.cfg
--- base/meta-raspberrypi/recipes-kernel/linux/files/container.cfg	1970-01-01 01:00:00.000000000 +0100
+++ modified/meta-raspberrypi/recipes-kernel/linux/files/container.cfg	2022-06-02 10:56:10.000000000 +0100
@@ -0,0 +1,14 @@
+CONFIG_CGROUP_PIDS=y
+CONFIG_CGROUP_DEVICE=y
+CONFIG_MEMCG=y
+CONFIG_BLK_CGROUP=y
+CONFIG_NAMESPACES=y
+CONFIG_UTS_NS=y
+CONFIG_IPC_NS=y
+CONFIG_USER_NS=y
+CONFIG_PID_NS=y
+CONFIG_NET_NS=y
+CONFIG_NETFILTER_XT_TARGET_LOG=y
+CONFIG_NETFILTER_XT_MATCH_PHYSDEV=y
+CONFIG_VETH=y
+CONFIG_CGROUP_FREEZER=y
diff -urN base/meta-raspberrypi/recipes-kernel/linux/linux-raspberrypi_5.4.bb modified/meta-raspberrypi/recipes-kernel/linux/linux-raspberrypi_5.4.bb
--- base/meta-raspberrypi/recipes-kernel/linux/linux-raspberrypi_5.4.bb	2022-11-08 09:10:58.874282984 +0000
+++ modified/meta-raspberrypi/recipes-kernel/linux/linux-raspberrypi_5.4.bb	2022-11-08 09:32:51.176395065 +0000
@@ -12,3 +12,4 @@
             file://powersave.cfg \
             file://android-drivers.cfg \
            "
+SRC_URI += "${@bb.utils.contains('DISTRO_FEATURES', 'dac', 'file://container.cfg', '', d)}"
diff -urN base/meta-raspberrypi/recipes-kernel/linux/linux-raspberrypi.inc modified/meta-raspberrypi/recipes-kernel/linux/linux-raspberrypi.inc
--- base/meta-raspberrypi/recipes-kernel/linux/linux-raspberrypi.inc	2022-11-08 09:10:58.874282984 +0000
+++ modified/meta-raspberrypi/recipes-kernel/linux/linux-raspberrypi.inc	2022-11-08 09:16:04.588160768 +0000
@@ -140,3 +140,5 @@
         echo "${CMDLINE}${PITFT_PARAMS}" > ${DEPLOYDIR}/${BOOTFILES_DIR_NAME}/cmdline.txt
     fi
 }
+
+CMDLINE_append += "${@bb.utils.contains('DISTRO_FEATURES', 'dac', 'cgroup_enable=cpuset cgroup_enable=memory cgroup_memory=1', '', d)}"
