diff --git a/recipes-rdkb/usp-pa/usp-pa.bb b/recipes-rdkb/usp-pa/usp-pa.bb
index 2b1df79..a313d57 100644
--- a/recipes-rdkb/usp-pa/usp-pa.bb
+++ b/recipes-rdkb/usp-pa/usp-pa.bb
@@ -17,8 +17,8 @@ SRC_URI = "https://github.com/BroadbandForum/obuspa/archive/v${OBUSPA_REL}-maste
 SRC_URI[obuspa.md5sum] = "b709d54f08aa528b73d435d71717f34f"
 
 # USPPA is the RDK specializations
-SRC_URI += "git://github.com/rdkcentral/usp-pa-vendor-rdk;protocol=http;branch=main;name=usppa"
-SRCREV_usppa = "c2a275ee27eb86a7df128ce790a50ef42cccc5da"
+SRC_URI += "git://git@github.com/rdkcentral/DSM.git;subpath=integration/rdk-b-22q1/usp/src;protocol=ssh;branch=main;name=usppa"
+SRCREV_usppa = "403a90c3f0997035f2f117a570a6c67ab4af48cc"
 
 # Patches for OBUSPA
 #SRC_URI += "file://patches/example.patch"
@@ -52,7 +52,7 @@ LDFLAGS += "-ldbus-1 -lccsp_common"
 
 # Specialize the OBUSPA release by copying across the RDK specific source files to the source directory
 do_configure_prepend() {
-    cp ${WORKDIR}/git/src/vendor/* ${S}/src/vendor
+    cp ${WORKDIR}/src/vendor/* ${S}/src/vendor
 }
 
 # Copy files to staging area
