diff -urN 22q3/meta-rdk/recipes-core/packagegroups/packagegroup-rdk-oss-broadband.bb modified/meta-rdk/recipes-core/packagegroups/packagegroup-rdk-oss-broadband.bb
--- 22q3/meta-rdk/recipes-core/packagegroups/packagegroup-rdk-oss-broadband.bb	2022-11-14 14:41:41.926656883 +0000
+++ modified/meta-rdk/recipes-core/packagegroups/packagegroup-rdk-oss-broadband.bb	2022-11-14 14:46:36.729047935 +0000
@@ -71,3 +71,6 @@
 
 RDEPENDS_packagegroup-rdk-oss-broadband += " ${@bb.utils.contains('DISTRO_FEATURES', 'snmppa', 'net-snmp-client net-snmp-server net-snmp-mibs net-snmp-server-snmpd', '', d)}"
 RDEPENDS_packagegroup-rdk-oss-broadband += " ${@bb.utils.contains('DISTRO_FEATURES', 'rdk-oss-ssa', 'rdk-oss-ssa ecryptfs-utils', '', d)}"
+
+RDEPENDS_packagegroup-rdk-oss-broadband += " ${@bb.utils.contains('DISTRO_FEATURES', 'dac', 'dobby', '', d)}"
+RDEPENDS_packagegroup-rdk-oss-broadband += " ${@bb.utils.contains('DISTRO_FEATURES', 'dac', 'dsm', '', d)}"
